ext/bcmath/libbcmath/src/int2num.c

```diff
@@ -55,7 +55,7 @@ void bc_int2num(bc_num *num, int val)
 	while (val != 0) {
 		*bptr++ = val % BASE;
 		val = val / BASE;
-		ix++; 		/* Count the digits. */
+		ix++;        /* Count the digits. */
```

Comment:
Ditto, might make more sense to either have it 1 space or move the comment above the increment line

Reply:
Moved the comments up.